plugins/identities/http.go

```diff
@@ -126,6 +137,13 @@ func (c *HTTPClientConfig) Decrypt() error {
 			}
 			c.Auth.CredentialsJSON = decryptedValue
 		}
+		if c.Auth.CredentialsJSONBase64 != "" {
+			decryptedValue, err := c.crypto.Decrypt(c.Auth.CredentialsJSONBase64)
```

Comment:
Can we here decode and check whether the encoded base64 is in `JSON` format? If not, return error early. This is to make sure that whatever data that we decrypt later will always be in JSON.

Reply:
added the validation in the `.Validate()` method instead here https://github.com/goto/guardian/pull/61/commits/a05c7f119d26d1064fbfdd177d4abfcf061f118a

and it's called here https://github.com/goto/guardian/blob/main/core/policy/service.go#L252 for policy creation